tgui/packages/tgui-say/handlers/componentUpdate.tsx

```diff
@@ -1,6 +0,0 @@
-import { Modal } from '../types';
-
-/** After updating the input value, sets back to false */
-export const handleComponentUpdate = function (this: Modal) {
-  this.setState({ edited: false });
```

Comment:
I was using this to trigger the inner textarea component to rerender whenever history was viewed or radio prefixes changed. it would flip on, rerender, then the tguisay would flip the switch back off. very hacky